app/containers/Bill/index.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable */
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+// Redux imports
+import { connect } from 'react-redux';
+import { createStructuredSelector } from 'reselect';
+import { makeGetOrders, makeGetAmount } from 'containers/Orders/selectors';
+import red from 'material-ui/colors/red';
+import blueGrey from 'material-ui/colors/blueGrey';
+import BillItem from './BillItem';
+import Button from 'material-ui/Button';
```

Comment:
this will crash because of the material migration, have you rebased your branch?

Reply:
Not yet.